src/frontend/src/sql.rs

```diff
@@ -91,21 +83,38 @@ pub(crate) fn insert_to_request(table: &TableRef, stmt: Insert) -> Result<Insert
         table_name,
         columns_values: columns_builders
             .into_iter()
-            .map(|(c, _, mut b)| (c.to_owned(), b.to_vector()))
+            .map(|(cs, mut b)| (cs.name.to_owned(), b.to_vector()))
             .collect(),
     })
 }
 
 fn add_row_to_vector(
-    column_name: &str,
-    data_type: &ConcreteDataType,
+    column_schema: &ColumnSchema,
     sql_val: &SqlValue,
     builder: &mut Box<dyn MutableVector>,
 ) -> Result<()> {
-    let value = statements::sql_value_to_value(column_name, data_type, sql_val)
-        .context(error::ParseSqlSnafu)?;
+    let value = if replace_default(sql_val) {
```

Comment:
Looks like these logics are almost the same with `insert.rs` in datanode, we can refactor them later. can you add a TODO here? Thanks.

Reply:
There is already a TODO in `frontend/src/sql.rs`. 

// TODO(fys): Extract the common logic in datanode and frontend in the future.

And maybe I can take that too after is PR.